package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "azure-sdk-for-js",
+  "author": "Microsoft Corporation",
+  "version": "0.1.0-preview",
+  "description": "Microsoft Azure Client Libraries for Javascript",
```

Comment:
Microsoft Azure Client Libraries for JavaScript with TypeScript type definitions

Reply:
Do people really read that?

Reply:
I expect people to largely follow these steps:

1. Install `@azure/resources` or a similar package
2. Think "well, how can I install the corresponding `@types` package?
3. Give up and just try to start using the package
4. Profit

Unlikely people will ever read the description in package.json.

Reply:
Going a little further--I'm not sure why there should be a root package.json if we don't ship a rollup package. Do you think there's a reason to have one?

Reply:
The only reason I kept it was to use it for its `scripts: {…}` section.